cli/render.js

```diff
@@ -54,5 +54,9 @@ export default (renderApp, options) => {
   // Every script tag that create-react-app inserts, make them defer
   $("body script[src]").attr("defer", "");
 
+  $("body script[src]").appendTo("head");
+
+  $("body script[src]").remove();
+
```

Comment:
This is awesome! 
nit: would you mind putting those two lines together since they're very related to each other. 
Also (not that I'm saint with my previous edits on this file) but would you mind writing a comment that explains why we're doing this. E.g.

```javascript
// Move the script tags from the body to the head. 
// That way the browser can notice, and start downloading these files sooner
// but they'll still be executed after the first render.
```

Reply:
😁 Alright, done!